topics/admin/tutorials/stuff/slides.html

```diff
@@ -1,29 +1,3 @@
----
-layout: tutorial_slides
-logo: "GTN"
-
-title: "Server Monitoring and Maintenance"
-zenodo_link: ""
-questions:
-  - How to monitor a Galaxy service?
-  - What are the best practices to maintain a Galaxy server?
-objectives:
-  - Learn about different monitoring strategies.
-  - Setup and start the Galaxy reports app.
-time_estimation: "1h"
-key_points:
-  - Galaxy supports pluggable monitoring extensions.
-  - Use grafana or the reports webapp to monitor your service.
-contributors:
-  - natefoo
-  - bgruening
-  - slugger70
----
-layout: true
-name: left-aligned
-class: left, middle
----
-
```

Comment:
This is missing metadata (not on purpose, I suppose).

Reply:
Somewhat on purpose, hence the name `stuff` as well, it was just incoherent random stuff extracted from monitoring/maintenance that weren't directly related. Most of it seems to fall under "admin of galaxy tips" but I don't know what to do with them, so stuck them in a 'stuff' folder under someone needed them and had time to make them into a coherent story. I'll re-add the metadata with `enabled: false`, but if you have any good ideas I'd love to hear them!